docs/tutorials/running-the-server.rst

```diff
@@ -0,0 +1,17 @@
+Running the HDF5 Reader Service server
+======================================
+
+This tutorial shows how to run the HDF5 Reader Service server.
+
+Start the server
+----------------
+
+It is very easy to start the HDF5 reader service. All that is needed is to
+run the following command in a terminal:
+
+    $ uvicorn hdf5_reader_service.main:app
```

Comment:
Now that https://github.com/DiamondLightSource/python-hdf5-reader-service/pull/12 has been merged this can be 

```suggestion
    $ hdf5-reader-service --host <host> --port <port>
```